version/info_go118.go

```diff
@@ -27,19 +28,24 @@ func getRevision() string {
 	return computedRevision
 }
 
+func getTags() string {
+	return computedTags
+}
+
 func init() {
-	computedRevision = computeRevision()
+	computedRevision, computedTags = computeRevision()
 }
 
-func computeRevision() string {
+func computeRevision() (string, string) {
 	var (
 		rev      = "unknown"
+		tags     = "unknown"
```

Comment:
Wouldn't this be `none` if we're building with no tags?

Reply:
I am fine with unknown, maybe this can be removed by the compiler? let's proceed as is.